base/setup/welcome/lang/hi-IN.rc

```diff
@@ -0,0 +1,40 @@
+/* Hindi translation by Arnav Bhatt */
```

Comment:
There's an official header style that you must abide. See -- http://reactos.org/wiki/Coding_Style#File_Structure

Reply:
So I need to remove it? @Fraizeraust

Reply:
Just replace your comment header at the top with the one from the link. Make sure some of the comments like in the front of `PROJECT` (ReactOS Kernel) must be replaced according to the specific module you've did your changes.